tensorflow/contrib/deprecated/__init__.py

```diff
@@ -18,42 +18,40 @@
 submodule, and made some semantic tweaks. The first thing to note is that we
 moved the APIs around as follows:
 
+```python
 tf.scalar_summary -> tf.summary.scalar
-
 tf.histogram_summary -> tf.summary.histogram
-
 tf.audio_summary -> tf.summary.audio
-
 tf.image_summary -> tf.summary.image
-
 tf.merge_summary -> tf.summary.merge
-
 tf.merge_all_summaries -> tf.summary.merge_all
+```
 
-We think this is a cleaner API and will improve long-term discoverability and
-clarity of the TensorFlow API. However, we also took the opportunity to make an
+We think this API is cleaner and will improve long-term discoverability and
+clarity of the TensorFlow API. We however, also took the opportunity to make an
```

Comment:
You either need another comma before 'however', or you revert this line. I'd go with the old order.